assets/js/components/Config/DeviceTags.vue

```diff
@@ -76,7 +76,7 @@ export default {
 				case "phasePowers":
 					return value.map((v) => this.fmtW(v, POWER_UNIT.KW, false)).join(" · ") + " kW";
 				case "chargeStatus":
-					return this.$t(`config.deviceValue.chargeStatus${value}`);
+					return this.$t(`config.deviceValue.chargeStatus${value || "F"}`);
```

Comment:
This is physically wrong. Maybe F, maybe anything else. Why should the status disappear? Use `-` instead?

Reply:
> If client disconnects evcc would return an error anyway

This is not the case. OCPP returned "" after client disconnected. If we can fix this at root level we should do.